app/models/user.rb

```diff
@@ -83,6 +83,7 @@ class User < ApplicationRecord
     s.string :mastodon_username
     s.any :keybase_signatures, array: true
     s.string :homepage
+    s.string :totp_recovery_codes, array: true, default: [], null: false
```

Comment:
I'm glad to have the `null: false`. Can you confirm that we don't need a migration? In a rails console, if you load a user, they'll automatically have an empty array? I don't want `authenticater_totp_recovery` to throw a `NoMethodError` on `nil`. Otherwise this will need a migration.